supertokens_python/recipe/thirdparty/utils.py

```diff
@@ -151,23 +159,47 @@ class ThirdPartyConfig:
     def __init__(self,
                  sign_in_and_up_feature: SignInAndUpFeature,
                  email_verification_feature: ParentRecipeEmailVerificationConfig,
-                 override: OverrideConfig):
+                 override: OverrideConfig,
+                 get_email_delivery_config: Callable[[RecipeImplementation], EmailDeliveryConfigWithService[TypeThirdPartyEmailDeliveryInput]],
```

Comment:
Input type should be RecipeInterface

Reply:
I have seen this type of error in other places too. Make sure to see all the recipes

Reply:
Fixed. Thanks!